extensions/wrapper/wrapper-api/src/main/java/de/sovity/edc/ext/wrapper/api/ui/model/ContractAgreementPageQuery.java

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (c) 2024 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ */
+
+package de.sovity.edc.ext.wrapper.api.ui.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Schema(description = "Data for querying a Contract Contract Agreement Page")
+public class ContractAgreementPageQuery {
+    @Schema(description = "Contract agreement status search criterion.")
+    private ContractTerminationStatus statusQuery;
```

Comment:
documentation makes no sense

I guess you want to have a field "Optionally limit resulting contract agreements to a given termination status", `terminationStatusFilter`

Reply:
> documentation makes no sense

Well, it's the criterion that will be applied to the status when querying the contract agreement.
But yes I take take the more explicit sentence.